src/main/java/Dude.java

```diff
@@ -0,0 +1,204 @@
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+
+public class  Dude {
+
+    static TaskList taskList = new TaskList();
```

Comment:
Consider to add the access modifier to the taskList as the variable are suggested code with `private` or `protected`.